yarn-project/aztec.js/src/account/index.ts

```diff
@@ -26,8 +26,8 @@ export function getEcdsaAccount(
   encryptionPrivateKey: GrumpkinPrivateKey,
   signingPrivateKey: Buffer,
   saltOrAddress?: Salt | CompleteAddress,
-): Account {
-  return new Account(rpc, encryptionPrivateKey, new EcdsaAccountContract(signingPrivateKey), saltOrAddress);
+): AccountManager {
```

Comment:
Should the function name convey that it is an account manager. I think it is fine as it is, just wanted to point it out.

Reply:
I had tried that change and reverted it. I agree it _should_ be a manager, but I also like it more as it is.